real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/PoseViewer.java

```diff
@@ -25,6 +29,7 @@
 	private ArrayList<Matrix> poses;
 	private JButton zoomIn, zoomOut;
 	private JSlider zoomMag;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'commtheta' could be replaced by a local variable.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2325208422/issues/source?bid=2938665&fileBranchId=3199623#l32)